### PR TITLE
Fix for issue 6: BeIDCards.getAllBeIDCards() contains cards in unrecoverable state after using the FedICT EID Viewer

### PR DESCRIPTION
**Pull request by dwagelaar**

### CODE REVIEW

**Comment by fcorneli** [comment]
Could you highlight a bit more which problem you exactly try to fix here?
We'll soon start working on Commons eID 0.7.0 and are in the process of determining the release scope.

File: commons-eid-bom/pom.xml [hunk 1]
```diff
@@ -1,12 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 	<modelVersion>4.0.0</modelVersion>
+	<parent>
```
Comment (fcorneli) [inline commons-eid-bom/pom.xml]: Reference to the parent POM within the BOM was not provided to keep the dependency management clean within the project where you import the BOM.
Also, AFAICR referring to the parent POM within the BOM made the maven-release-plugin to freak out.

Comment (dwagelaar) [inline commons-eid-bom/pom.xml]: That commit is not really required, and just made it easier to propagate a 'man versions:set' to all maven modules (we're using internal releases for the moment with our fixes applied). I will roll it back.

**Comment by dwagelaar** [comment]
> Could you highlight a bit more which problem you exactly try to fix here?

This pull request includes an earlier pull request by Peter Mylemans, simply because it's on the same branch. The additional commits try to fix various issues with accessing eID cards, issue #6 most notably.

The additional logging is mainly there for us to do a post-mortem analysis on application logs, such that we can find out when card/terminal events stopped working:
- [smart card stack on Mac OS X has many problems since Yosemite](https://ludovicrousseau.blogspot.be/2014/12/os-x-yosemite-and-smart-cards-known-bugs.html)
- some card readers appear to have unreliable firmware and stop working after a while, requiring the application to restart to get the reader working again

**Pull request merged**